nncf/torch/nncf_network.py

```diff
@@ -716,13 +718,15 @@ def _collect_eval_op_scopes(self, model: nn.Module, dummy_forward_fn: Callable)
         return result
 
     def get_node_to_op_address_mapping(self) -> Dict[NNCFNodeName, OperationAddress]:
-        # The IDs of corresponding nodes of the original dynamic graph and original NNCF graph
```

Comment:
Add docstring and typehint for return value

Reply:
Done